### DISCUSSION TITLE
Does React Native support string transforms?

### DISCUSSION THREAD

**Discussion by natew** [necolas/react-native-web#2510] [Q&A]
I noticed this commit:

https://github.com/necolas/react-native-web/commit/831f59e003203854b0320656bbfca418e29fac3e

But React Native doesn't document supporting string transforms:

https://reactnative.dev/docs/transforms

Is this undocumented or divergence from RN?

**Comment by necolas** [comment]
It was added to 0.71 https://github.com/facebook/react-native/issues/34425

And is documented here
https://reactnative.dev/docs/transforms#transform
